fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/resources/Resource.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.fiat.model.resources;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+
+public enum Resource {
+  ACCOUNT,
+  APPLICATION;
+
+  // TODO(ttomsu): This is Redis-specific, so it probably shouldn't go here.
+  public static Resource parse(@NonNull String key) {
```

Comment:
I'm willing to leave it here for now, just left the TODO as a reminder to maybe find a better place for it in the future, if it ends up being necessary.